exporter_test.go

```diff
@@ -1877,3 +1877,227 @@ func TestParseSentinelMasterString(t *testing.T) {
+	}
+	for _, tst := range tsts {
+		t.Run(tst.name, func(t *testing.T) {
+			chM := make(chan prometheus.Metric)
+			go func() {
+				e.processSentinelSlaves(chM, tst.slaveDetails, tst.labels...)
+				close(chM)
+			}()
+			want := map[string]bool{
+				"sentinel_master_ok_slaves": false,
+			}
+
+			for m := range chM {
+				for k := range want {
+					if strings.Contains(m.Desc().String(), k) {
+						want[k] = true
+						got := &dto.Metric{}
+						m.Write(got)
+
+						val := got.GetGauge().GetValue()
+						if int(val) != tst.expectedMetricValue[k] {
+							t.Errorf("Expected Metric value:%d didn't match to reported value:%d for test:%s  ", tst.expectedMetricValue[k], int(val), tst.name)
```

Comment:
```suggestion
							t.Errorf("Expected metric value %d didn't match to reported value %d for test %s", tst.expectedMetricValue[k], int(val), tst.name)
```

Reply:
This was updated along with other review suggestion. Thanks. :)